### DISCUSSION TITLE
Paste from system clipboard

### DISCUSSION THREAD

**Discussion by System3-2** [LazyVim/LazyVim#923] [Q&A]
I reinstalled my arch linux due to a break in my  installation. and i reinstalled lazyvim, the problem now is i can paste from system clipboard to neovim, which i was able to do before.

**Comment by jyuan0** [comment]
This isn't related to LazyVim, but you probably want to add/set `unnamedplus` to the `clipboard` option, and have the appropriate tool installed:

- X: `xsel`, or `xclip`
- Wayland: `wl-copy`, `wl-paste`

See `help clipboard` and `help 'clipboard'`.

**Reply by folke** [reply]
not need to change clipboard options, those are already the default for LazyVim.

missing one of those tools is most likely the issue indeed

**Reply by ChenZhongPu** [reply]
But recently, it would show a message when to copy something:

> clipboard: error invoking xclip: Authorization required.
